### DISCUSSION TITLE
怎么增加对fi这种连字符处理。

### DISCUSSION THREAD

**Discussion by earthmoon** [xiaoyifang/goldendict-ng#1050] [Q&A]
您好，

在处理fi这种连字符时，goldendict抓取的单词不正确，中间会多一个奇怪的字母。比如：

![image](https://github.com/xiaoyifang/goldendict-ng/assets/45897471/c3f5e461-5584-4eaa-894d-57198428d527)

x'x谢谢。

**Comment by xiaoyifang** [comment]
pdf吗，你拷贝之后，复制到word或者文本编辑器里面看下。

**Reply by earthmoon** [reply]
![image](https://github.com/xiaoyifang/goldendict-ng/assets/45897471/0151bd15-50e4-4ed5-94b0-5a4251c47335)


这个样子，也是没法区分。

**Comment by xiaoyifang** [comment]
pdf制作的问题。
https://stackoverflow.com/a/55491402/968188
https://community.adobe.com/t5/acrobat-discussions/text-encoding-errors-when-copying-and-pasting-from-pdf/m-p/12042119

**Reply by earthmoon** [reply]
有办法绕过这个问题吗？

**Comment by xiaoyifang** [comment]
通过ocr工具识别后，调用goldendict.
https://xiaoyifang.github.io/goldendict-ng/howto/ocr/

**Reply by earthmoon** [reply]
谢谢，可以把这个功能直接集成到下一个版本吗？

**Reply by xiaoyifang** [reply]
没有计划，  ocr的工具可以使用任意的ocr工具。不局限于capture2text.

**Reply by earthmoon** [reply]
是我理解错了吗，OCR不是识别图像的吗？怎么用来识别这个了，难道要把这个当成图像处理吗？谢谢。

**Reply by xiaoyifang** [reply]
是的。

**Reply by earthmoon** [reply]
好吧，谢谢。
